gen/internal/config/config.go

```diff
@@ -81,6 +81,8 @@ type QueryConfig struct {
 	// a pointer type would just be annoying for client code, but sometimes you
 	// do actually want nullable arguments.
 	NullableArguments bool `toml:"nullable_arguments"`
+	// If true, blah blah blah
```

Comment:
This comment still needs to be updated.

Reply:
I tried to leave a good comment here but lining up my wording to the other definitions is a little hard so feel free to change it or if I need to add more details let me know.

Reply:
Your description looks good to me :)